### ISSUE TITLE
Fibro-obliterative bile-duct lesion

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7554]
**Preferred term label:**
Fibro-obliterative bile-duct lesion

**Synonyms**
Fibro-obliterative loss of bile ducts
Fibro-obliterative bile-duct scarring

**Definition (free text, please give PubMed ID)**
A condition in which the lumen of a bile duct has been filled by fibrous scar material, in effect replacing the bile duct with a fibrous scar.
See Figure 3 in PMID:18701947

**Parent term (use hpo.jax.org/app)**
Abnormality of the intrahepatic bile duct HP:0011040

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
619849 
PMID: 32737136  3/0

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
